### PR TITLE
Implement fixes for my favorite pet peeves in Vanilla

### PR DESCRIPTION
**Pull request by ddv239**
Populate missing Artist/AlbumArtist values from each other.
Make sure that repeatedly playing the queue to completion with the "No repeat" setting plays it again completely, not just the last track.

### CODE REVIEW

File: res/values-ru/translatable.xml [hunk 1]
```diff
@@ -57,35 +57,33 @@
   <string name="open">Открыть</string>
   <string name="go_home">Начальная папка</string>
   <string name="jumping_to_song">Дорожка уже в очереди, следующий…</string>
+  <!--See http://unicode.org/reports/tr35/tr35-numbers.html#Language_Plural_Rules and
```
Comment (adrian-bl) [inline res/values-ru/translatable.xml]: We are using Transifex for all translations, as described here: https://github.com/vanilla-music/vanilla#translating

Could you please revert this and add your translations in transifex? They will get included in the build the next time i trigger a sync.

File: src/ch/blinkenlights/android/medialibrary/MediaMetadataExtractor.java [hunk 2]
```diff
@@ -369,6 +369,14 @@ private void populateSelf(HashMap bastp) {
 			}
 		}
 
+		// If only one of (ARTIST, ALBUMARTIST) is present, populate the other
+		if (!containsKey(ARTIST) && containsKey(ALBUMARTIST)) {
+			put(ARTIST, get(ALBUMARTIST));
+		}
+		if (containsKey(ARTIST) && !containsKey(ALBUMARTIST)) {
+			put(ALBUMARTIST, get(ARTIST));
+		}
```
Comment (adrian-bl) [inline src/ch/blinkenlights/android/medialibrary/MediaMetadataExtractor.java]: note to self: we should trigger a library rescan after this change was submitted.

File: res/values-ru/translatable.xml (hunk 1 above)
Comment (ddv239) [reply res/values-ru/translatable.xml]: Will do.

Comment (ddv239) [reply res/values-ru/translatable.xml]: I requested joining the Transifex project.

**Comment by ddv239** [comment]
I rolled back the translation change and will submit it separately later.

**Comment by adrian-bl** [comment]
thanks! I updated the translations in 0be6bbf85326bcf80babfa57ca5057c50f58624b

**Pull request merged**